tests/src/JIT/Methodical/doublearray/dblarray3.cs

```diff
@@ -99,7 +99,7 @@ public static void f2b()
     public static void f3()
     {
         Array arr = Array.CreateInstance(typeof(double), 1000);
-        if (GC.GetGeneration(arr) != 0)
+        if (GC.GetGeneration(arr) != s_LOH_GEN)
```

Comment:
It looks like some allocation helpers did not check `DoubleArrayToLargeObjectHeapThreshold`. I think that was not intentional.